projects/reth-pureth-meta.md

```diff
@@ -0,0 +1,152 @@
+# Pureth (EIP-7919) proposal for Reth
+
+## Introduction
+The [Pureth proposal](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-7919.md) contains a bunch of EIPs aimed at making Ethereum accessible without the need of trusted RPCs. This is crucial to Ethereum's principles, as untrusted RPCs could make the whole network trustless for constrained devices which can't run a full node themselves. The proposal aims to make Ethereum accessible to resource-constrained devices in a trustless manner.
+Upon talking with my mentor, Etan about this proposal, this proposal needs to be divided into two main parts. One is implementing the [2D logs filter structure](https://eips.ethereum.org/EIPS/eip-7745) and the other is SSZ types, mainly [SSZ Progressive List](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-7916.md). I have chosen to take up implementing the 2D logs filter in Reth.
+
+## 2D Logs Filter
+
+My project will focus mainly on implementing the 2D logs filter (EIP-7745) in the Reth execution client.
+
+### Motivation
+The current Bloom Filter data structure is [highly inefficient](https://ethereum-magicians.org/t/eip-7668-remove-bloom-filters/19447) as it gives a lot of false positives and it takes a lot of computation. It is useless to the point where it should be removed and replaced by a more efficient structure. [Zsolt](https://github.com/zsfelfoldi) proposed a global log value index, which is mapped into a 2D filter map, which allows for efficient querying and efficient validity proofs.
+
+### Importance
+A correct implementation of this structure should result in large speedups of querying logs and should result in significantly lower false positive rates.
+
+### Project Description
+This project aims to implement the EIP-7745 logs filter structure in Reth according to the [EIP specifications](https://eips.ethereum.org/EIPS/eip-7745) and similar to [Geth's implementation](https://github.com/zsfelfoldi/go-ethereum/blob/proof-poc/core/filtermaps) of the structure.
+
+### Specification
+The Pureth meta focuses on the Execution layer mainly, with some changes to the consensus layer.
+
+#### Consensus Layer changes
```

Comment:
Is this relevant for implementation in Reth? I understand reth will expect a different header but the spec of your project should focus on actual implementation